src/comm.cpp

```diff
@@ -934,6 +935,20 @@ void game_loop(int mother_desc)
       }
     }
 
+    {
+      // Send GMCP Vitals
+      for (d = descriptor_list; d; d = next_d) {
+        next_d = d->next;
+        if (!d->prompt_mode) continue;
+        if (!d->character) continue;
+
+        if (d->character->in_room) {
```

Comment:
Why only if they're in a room? Are you invoking room-specific things in vitals/pools?